examples/ordering/run.sh

```diff
@@ -1,29 +1,88 @@
 #!/bin/bash
 
-if [ "$#" -ne 1 ]; then
-    echo "Usage: run.sh n_members"
+usage() {
+    echo "Usage: ./run.sh [-n N_NODES] [-m N_MALFUNCTIONING_NODES] [-c N_CRASHES] [-o N_ORDERED_PER_CRASH] [-d RESTART_DELAY]"
     exit 1
-fi
+}
+
+N_NODES=5
+N_MALFUNCTIONING_NODES=5
```

Comment:
Maybe faster default example would be to have 3 normal nodes and 1 malfunctioning? Or something where 2/3+1 nodes are alive, so finalization goes on? Just a suggestion

Reply:
but then we might hit the problem with lack of censorship resistance...

I'd leave it that way so "it works"...

Reply:
fixed